spec/javascripts/packages/document-capture/higher-order/with-background-encrypted-upload-spec.jsx

```diff
@@ -16,29 +51,128 @@ describe('withBackgroundEncryptedUpload', () => {
     return null;
   });
 
+  describe('encrypt', () => {
+    it('resolves to AES-GCM encrypted data from string value', async () => {
+      const key = await window.crypto.subtle.importKey(
+        'raw',
+        new Uint8Array(32).buffer,
+        'AES-GCM',
+        false,
+        ['encrypt', 'decrypt'],
+      );
+      const iv = new Uint8Array(12);
+      const data = 'Hello world';
+      const expected = new Uint8Array([
+        134,
```

Comment:
this seems like we could just put these all on one line and add an eslint ignore?

Reply:
> this seems like we could just put these all on one line and add an eslint ignore?

I don't know that we can convince Prettier to ignore it (editor format-on-save integrations, for example).

Should be possible to just combine it into a string and `'134,...'.split(',')` though.

Reply:
> Should be possible to just combine it into a string and `'134,...'.split(',')` though.

Updated in 588e96d. Not a huge fan of the line arrangement for the `split` call, but at least it's fewer lines total 😄